cc_licenses/templates/includes/use_of_licenses.html

```diff
@@ -1,37 +1,43 @@
-
+{% load i18n %}
 
 {# For JS Toggling Feature for Consideration Sections - See licenses_detail.html #}
 <div id="about-cc-and-license" class="padding-bigger margin-top-bigger has-background-info-light" >
-  <h4 class="level is-vcentered b-header">Using Creative Commons Public Licenses</h4>
+  <h4 class="level is-vcentered b-header">{% trans "Using Creative Commons Public Licenses" %}</h4>
   <p class="has-text-black body-big">
-    Creative Commons public licenses provide a standard set of terms and conditions that creators
-    and other rights holders may use to share original works of authorship and other material subject
-    to copyright and certain other rights specified in the public license below. The following considerations
-    are for informational purposes only, are not exhaustive, and do not form part of our licenses.
+    {% blocktrans trimmed %}
+      Creative Commons public licenses provide a standard set of terms and conditions that creators
+      and other rights holders may use to share original works of authorship and other material subject
+      to copyright and certain other rights specified in the public license below. The following considerations
+      are for informational purposes only, are not exhaustive, and do not form part of our licenses.
+    {% endblocktrans %}
   </p>
   <hr class="divider rtl-divider">
   <div>
-    <h4 class="level is-vcentered b-header">Considerations for licensors<span class="level-right"><i class="angle-down icon" data-consideration="1" data-direction="down"><span class="is-sr-only">Show Considerations for Licensors</span></i></span></h4>
+    <h4 class="level is-vcentered b-header">{% trans "Considerations for licensors" %}<span class="level-right"><i class="angle-down icon" data-consideration="1" data-direction="down"><span class="is-sr-only">{% trans "Show Considerations for Licensors" %}</span></i></span></h4>
     <p class="has-text-black body-big is-hidden">
+    {% blocktrans trimmed %}
       Our public licenses are intended for use by those authorized to give the public permission to use material in ways otherwise
       restricted by copyright and certain other rights. Our licenses are irrevocable. Licensors should read and understand the terms
       and conditions of the license they choose before applying it. Licensors should also secure all rights necessary before applying
       our licenses so that the public can reuse the material as expected. Licensors should clearly mark any material not subject to the license.
       This includes other CC-licensed material, or material used under an exception or limitation to copyright.
       <a href="#">More considerations for licensors.</a>
+    {% endblocktrans %}
     </p>
   </div>
   <hr class="divider {% if is_rtl %}rtl-divider{% endif %}">
```

Comment:
We need to update this section. `rtl-divider` is not being toggled on an off. The divider is misaligned when a `rtl` language is displayed.

Reply:
I'm changing the CSS a little so .divider does the right thing based on some ancestor having [dir="ltr"] or [dir="rtl"]. 

I'm a little concerned that this won't work if somehow we haven't set [dir=] in the page, but I guess we can assume it or lots of things would stop working right.

Reply:
I like the idea of using django. We are trying to keep all css to a minimum that not's bulma.io or vocabulary. I also made sure to ask the cc-vocabulary team before adding the custom css you see in the templates. can we do something like 
```python
{% load get_language_bidi as lang_bidi %} 

{% if lang_bidi %}rtl-divider{% endif %}